doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -64,7 +64,13 @@ Bluetooth LE
 Bluetooth mesh
 --------------
 
-|no_changes_yet_note|
+* Fixed:
+
+  * An issue in :ref:`bt_mesh_light_ctrl_srv_readme` model where multiple scene recall messages for the same scene will not repeately trigger same scene recall. This helps in preventing ongoing transition interruption.
+
+* Updated:
+
+  * :ref:`bt_mesh_light_ctrl_srv_readme` model to make sure when controller is enabled and disabled, the regulator starts running when fresh value of ambient lux level is reported.
```

Comment:
Not sure I understand the sentence...

Reply:
Hmm, 'disabled' is probably not needed. How about this?
`... make sure when the controller is enabled, the regulator starts running when a fresh value of ambient lux level is reported.`

Reply:
Also, is it clear what regulator we are talking about, or should we state illuminance regulator?

Reply:
And should it then be LuxLevel or still lux level?
Ref. the other comment in the light_ctrl_srv.c file.